task_DEV_6/task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class CustomStack
+    {
+        private const int ARRAY_LENGTH = 10;
+        private const string INVALID_RANGE_ERROR_MESSAGE = "There are no elements.";
+
+        private int[] elements;
+        private int count;
+
+        public CustomStack()
+        {
+            elements = new int[ARRAY_LENGTH];
+        }
+
+        public CustomStack(int capacity)
```

Comment:
If you use variable `LENGTH` above you need to use `length` here too instead of `capacity` for consistency.

Reply:
Done.